fabtests/functional/cm_data.c

```diff
@@ -216,13 +216,28 @@ static int server_accept(size_t paramlen)
 	if (ret)
 		goto err;
 
-	FT_CLOSE_FID(ep);
-	FT_CLOSE_FID(rxcq);
-	FT_CLOSE_FID(txcq);
-	FT_CLOSE_FID(rxcntr);
-	FT_CLOSE_FID(txcntr);
-	FT_CLOSE_FID(av);
-	FT_CLOSE_FID(domain);
+
+	ret = ft_close_fid(ep);
+	if (ret)
+		return ret;
+	ret = ft_close_fid(rxcq);
+	if (ret)
+		return ret;
+	ret = ft_close_fid(txcq);
+	if (ret)
+		return ret;
+	ret = ft_close_fid(rxcntr);
+	if (ret)
+		return ret;
+	ret = ft_close_fid(txcntr);
+	if (ret)
+		return ret;
+	ret = ft_close_fid(av);
+	if (ret)
+		return ret;
+	ret = ft_close_fid(domain);
+	if (ret)
+		return ret;
```

Comment:
The cm_data tests hits a segfault because it tries to close the ep (and other fid's) twice on the server side.  This is a result of the fid's not being set to null by the above calls.